docs/overview/introduction.rst

```diff
@@ -0,0 +1,21 @@
+What It Is
+==================
+
+*CERN Analysis Preservation* (CAP) is a service to capture, preserve and reuse analysis information in HEP (high energy physics).
+It is developed by the collaborative effort of information and computer scientists at CERN, in partnership with the four largest LHC collaborations. A pre-release version is available to the latter for limited use.
+
+The initial idea behind the project was to preserve analyses for the purpose of **reproducible research**, making it accessible, understandable and reusable in many years to come. In conversations with LHC physicists it became apparent that the information we collect is valuable not only in years to come but already from the very start of information taking.
+
+Making analysis information accessible within the working group or collaboration opens up new possibilities for understanding, searching for and working with this information some of which are described in the :doc:`next section <project>`. The idea is to make **collaborative research** easier and increase the impact of preserved analyses.
+
+To us, an **analysis** consists of both data (e.g. datasets, code, results) and meta data (e.g. analysis name, contact persons, publication) the combination of which we call analysis information. While we structure this information on CAP in a way that represents the analysis workflow steps, we do not require or ask for any change in the physicists individual workflow or the terminologies used in different collaborations and working groups.
```

Comment:
I would like to add a sentence reflecting the discussion from the workshop around analysis content/workflows which change over time and that's why we version schemas